turbinia/api/schemas/request_options.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Turbinia API server request options model and methods."""
+
+from typing import Optional, List
+from pydantic import BaseModel, Field
+
+
+class BaseRequestOptions(BaseModel):
```

Comment:
Do we need objects here for the other evidence types in the evidence enum?  Or does it use the `BaseRequestOptions` for that?

Reply:
This has since been refactored to make use of the existing serialization methods in the evidence module.